.github/workflows/ci.yaml

```diff
@@ -125,6 +125,7 @@ jobs:
         os_version:
           - 15.5
```

Comment:
Leap 15.5 is EOL since End of December 2024. Let's remove it as well.

Reply:
good point, added another commit